qgis-projects/README.md

```diff
@@ -0,0 +1,141 @@
+
+Linux and Mac users can create a symbolic link using the ```ln``` command, e.g. ```ln -s /home/user/data/project ./data```
+
+Windows users may use the ```mklink``` function from *cmd*, e.g. ```mklink /D ../../../data ./data```.
+
+### Project data
+
+Adding data should be done directly by primary project custodians wherever possible. Adding of features or layers to existing data sources (remote/ gpkg etc) should by distributed in the same manner as the original data source (republished) or have their source details modified in the following section.
+
+---
+
+- File: ./data/dominode_data.gpkg
+- Custodian: Default
+- Description: Local copy of all data directly copied from remote staging database. It is recommended that this is copied directly from the Dominode staging environment database.
+- Source: Connect to the database, create a local geopackage named dominode_data.gpkg and select all features, then drag and drop them onto the gpkg within the QGIS Browser. Contact the custodian for database connection details.
+ 
+---
+
+- File: ./data/auxillary_data.gpkg
+- Custodian: Default
+- Description: Additional data required but not present in core database. This data should be accounted for in the remote data source later on.
+- Source: [Download from google drive](https://drive.google.com/file/d/1lH3dr4Ic3EZmHfmmZXPCjM-vbp6V6epN/view?usp=sharing)
+
+---
+
+- File: ./local/COMPASS_ROSE.geojson
+- Custodian: Default
+- Description: Feature generated with the Magnetic Declination QGIS plugin
+- Source: Contact custodian
+
+---
+
+> Note that the *data* directory is intended to be a symbolic link to a local data directory separate from the project repository.
+
+## File locks
+
+To prevent unwanted/ implicit changes to map projects, custodians may modify the file permissions for map projects to "read-only" prior to performing a commit.
+
+## Directory structure
+
+Primary custodians for the qgis-projects directory should be indicated in the ```CODEOWNERS``` in the *Project Root*.
+
+```text
+qgis-projects/    @user01 @user02
+```
+
+Additional ownership can be added to the ```CODEOWNERS``` file where relevant.
+
+```text
+qgis-projects/file-name.qgs    @user03
+```
+
+The following lines should be added to ```.gitignore``` in the *Project Root*.
+
+```text
+qgis-projects/data
+qgis-projects/**/*.qgs~
+qgis-projects/**/*.qgz~
+```
+
+This prevents git from interfering with the symbolic link to the data directory, which is likely to differ between users and operating systems. It also prevents the inclusion of QGIS autogenerated "backup" files, which are largely unnecessary when utilising a distributed VCS such as git.
+
+The following lines should be added to ```.gitattributes``` in the *Project Root*.
+
+```text
+qgis-projects/**/*.qgs lockable
+qgis-projects/**/*.qgz lockable
+```
+
+This ensures that QGIS project files are made available for *locking* by the Git LFS system.
+
```

Comment:
This also does not apply anymore, since we decided not to use git locking for now